infra/main.bicep

```diff
@@ -123,12 +123,17 @@ param useCommonAppServicePlan string = 'auto'
 var prefix = '${environmentName}-${environmentType}'
 
 // Boolean to indicate the various values for the deployment settings
-var isMultiLocationDeployment = secondaryAzureLocation == '' ? false : true
+var isMultiLocationDeployment = azureSecondaryLocation == '' ? false : true
 var isProduction = environmentType == 'prod'
 var isNetworkIsolated = networkIsolation == 'true' || (networkIsolation == 'auto' && isProduction)
-var willDeployHubNetwork = isNetworkIsolated && (deployHubNetwork == 'true' || (deployHubNetwork == 'auto' && !isProduction))
+var willDeployHubNetwork = isNetworkIsolated && (deployHubNetwork == 'true' || (deployHubNetwork == 'auto' && isProduction))
```

Comment:
Good catch! Does the next line need this same fix?

Reply:
Not currently. As-is, the defaults for `deployHubNetwork` and `useCommonAppServicePlan` are set to 'auto' so we want different behaviors.

- In dev we should have 1 app service plan, in prod we should have two app service plans.
- In dev we should not deploy the hub, in prod we should deploy the hub (and other features)